.github/workflows/tests.yml

```
@@ -140,6 +140,8 @@ jobs:
       - name: Run Tests
         # Windows is sensitive to long paths, using `--basetemp=${{ runner.temp }} to
         # keep the test directories shorter
+        env:
+          PYTHONPATH: .
         run: >
           pytest
```

AFAIK this is the equivalent to `python -m pytest` (see https://docs.pytest.org/en/stable/how-to/usage.html#calling-pytest-through-python-m-pytest), can we use that instead? I think `python -m` has greater precedence, will be less magical to new devs, and less likely to be reverted in the future (thinking about future Ken trying to cleanup and simplify stuff)?

```suggestion
        run: >
          python -m pytest
```

---

It won't have an effect for the subprocess that run `conda create` because those end up doing something akin to `python path/to/conda-entry-point` which doesn't inherit the parent `sys.path` nor adds `.` itself. So I think we do need `PYTHONPATH` :(

---

in that case this sounds more like something we can solve with an autouse fixture?

---

Done!